bin/lb-ng

```diff
@@ -26,9 +27,10 @@ assertLoopBackVersion();
 
 var ngModuleName = argv['module-name'] || 'lbServices';
 var apiUrl = argv['url'] || app.get('restApiRoot') || '/api';
+var includeSchemas = argv['include-schema'];
 
-console.error('Generating %j for the API endpoint %j', ngModuleName, apiUrl);
-var result = generator.services(app, ngModuleName, apiUrl);
+console.error('Generating %j for the API endpoint %j', ngModuleName, apiUrl, includeSchemas);
+var result = generator.services(app, ngModuleName, apiUrl, includeSchemas);
```

Comment:
@akera-io  Could you please rework this line to use the new options-based API introduced in https://github.com/strongloop/loopback-sdk-angular/pull/223?